backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/GitMacheteStatusTest.java

```diff
@@ -198,7 +198,7 @@ private void printCommonParts(BaseGitMacheteBranch branch, int level, StringBuil
           Case($(Ahead), "ahead of " + syncToRemote.getRemoteName()),
           Case($(Behind), "behind " + syncToRemote.getRemoteName()),
           Case($(Untracked), "untracked"),
-          Case($(Diverged), "diverged from " + syncToRemote.getRemoteName())));
+          Case($(DivergedAndNewerThanRemote), "diverged from " + syncToRemote.getRemoteName())));
```

Comment:
Case for `DivergedAndOlderThanRemote` missing? Plus add such a branch to the test repo setup script